docs/quickstart/set-up-a-connection.md

```diff
@@ -42,3 +42,10 @@ This is just the beginning of using Airbyte. We support a large collection of so
 If you have any questions at all, please reach out to us on [Slack](https://slack.airbyte.io/). We’re still in alpha, so if you see any rough edges or want to request a connector you need, please create an issue on our [Github](https://github.com/airbytehq/airbyte) or leave a thumbs up on an existing issue.
 
 Thank you and we hope you enjoy using Airbyte.
+
+
+{% hint style="warning" %}
+At the moment Airbyte runs a full-refresh to recreate the final tables. This can cause more costs in some destinations like Snowflake, Redshidt, Bigquery.
```

Comment:
```suggestion
At the moment, Airbyte runs a full-refresh to recreate the final tables. This can cause more costs in some destinations like Snowflake, Redshift, and BigQuery.
```